Js date picker

greywidget opened the pull request:
I used an older version of jquery/ui (1.8.24) to be compatible with the current local jquery_min.js (1.4.2)
The jquery/ui documentation says 1.8.24 is compatible with  jQuery 1.3.2 and newer.

Tested the new js calendar and existing booksearch on the following browser versions:
- Google Chrome Version 86.0.4240.75 (Official Build) (64-bit)
- Microsoft Edge Version 86.0.622.43 (Official build) (64-bit)
- Safari Version 12.1.2 (14607.3.9)

bbelderbos commented:
Sweet, doing my books in safari this bug annoys the heck out of me. Please put the js in the central script.js and I will merge and deploy.

Thanks!
Bob

greywidget commented:
Moved the js into script.js as requested.
Changed input_type from 'date' to 'text' to disable the browser datepicker from running on Firefox which was running both the js and the built_in datepickers.

bbelderbos commented:
Great, I will get this deployed later, thanks
